framework/wazuh/agent.py

```diff
@@ -1894,14 +1894,22 @@ def get_outdated_agents(offset=0, limit=common.database_limit, sort=None):
         return data
 
 
-    def _get_versions(self, wpk_repo=common.wpk_repo_url):
+    def _get_versions(self, wpk_repo=common.wpk_repo_url, desired_version=None):
         """
         Generates a list of available versions for its distribution and version.
         """
-        if self.os['platform']=="windows":
-            versions_url = wpk_repo + "windows/versions"
+        invalid_platforms = ["darwin", "solaris", "aix", "hpux"]
+
+        if self.os['platform'] in invalid_platforms or self.os['platform'] = "sles" and self.os['major'] == 11:
```

Comment:
self.os['platform'] == "sles"

Reply:
Hi @chemamartinez, 

this check is to avoid the installation of the WPK in a SUSE 11 only. The WPK works fine if you installed it in a SUSE 12.

I would add a check to avoid the installation of the WPK also in a CentOS 5/RHEL 5.

Reply:
Done in f47dea0